### ISSUE TITLE
This document format is not supported

### ISSUE THREAD

**Issue by crides** [tangrs/nspire-linux-loader2#8]
![03-26-2017 image 002](https://cloud.githubusercontent.com/assets/15631555/24331229/76dab988-1262-11e7-84c2-3af3790d9ec5.jpg)
My device is TI-nspire CX, and I just upgraded my nspire OS. But when I download the binary file from TI-planet ([The link](tiplanet.org/nspire-linux-builds/linuxloader2.tns)) and send it to my calc and open it, it shows the dialog in the picture.

**Comment by crides** [comment]
The link is https://tiplanet.org/nspire-linux-builds/linuxloader2.tns

**Comment by hazelduvall** [comment]
Have you first run the ndless installer? If not, and you don't even have it yet, it can be found here: http://ndless.me

**Comment by crides** [comment]
No, and now the file is loadable. But after I ran the linuxloader, the system reboots, and the Ndless installation is not in effect again (I mean, the message at the top disappeared, and the "unsupported" problem shows again).

**Comment by hazelduvall** [comment]
Non-persistence is a limitation of Ndless; you need to run the installer after every reboot. As for linuxloader not working, are you loading the kernel and everything before your calc reboots, or is it just crashing straight away? Also, what OS do you have?

**Comment by crides** [comment]
By saying "loading", do you mean typing and running the commands with instructions from [here](https://hackspire.org/index.php/Linux)? I cannot, it just makes the system reboot immediately after I ran it. For the OS, I just upgraded it, and the version is "4.4.0.532". Btw, it has Boot1 and Boot2 version numbers in the "about" section, which are "3.0.0.99" and "4.4.0.8" respectively.

**Comment by crides** [comment]
Well, I don't know why, but it suddenly works. But I just wanna ask one more question. In the link above, there is the zImage which is the kernel, but where can I find the initrd file?

**Comment by hazelduvall** [comment]
I believe they can be found at https://tiplanet.org/nspire-linux-builds/, same as the kernel. Look for the buildroot stuff

**Comment by crides** [comment]
Yes, I've seen those packages. But the file format should be `.tns`, and what it provides is a `.tar.xz` package. Is there some modification I need to do, like adding the `.tns` extension? That's what I'm curious about.
